open3d_conversions/src/open3d_conversions.cpp

```diff
@@ -15,9 +15,45 @@
 // C++
 #include <memory>
 #include <string>
+#include <sstream>
 
+#include <sensor_msgs/image_encodings.hpp>
 #include "open3d_conversions/open3d_conversions.hpp"
 
+// This is the best we have prior to C++20 std::endian::native
+static bool isLittleEndian(){
```

Comment:
A few tests for all these functions like in https://github.com/ros-perception/perception_open3d/blob/7cdffdfaa22f061b7652e4d2d6c34a39fdc9bea9/open3d_conversions/test/test_open3d_conversions.cpp are necessary to make sure there aren't any issues